packages/jellyfish-api-core/__tests__/category/blockchain.test.ts

```diff
@@ -314,4 +314,35 @@ describe('masternode', () => {
       })
     })
   })
+
+  describe('getBlockStats', () => {
+    it('should get blockchain stats and return  all values', async () => {
+      const blockHash  = await waitForBlockHash(1);
+      const stats = await client.blockchain.getBlockStats(blockHash);
+
+
+      expect(stats.height).toBeGreaterThanOrEqual(1);
+      expect(stats.minfee).toBeLessThanOrEqual(stats.medianfee);
+      expect(stats.medianfee).toBeLessThanOrEqual(stats.maxfee);
+    })
+
+    it('should  get blockchain stats with specific values', async () => {
+      const blockHash  = await waitForBlockHash(1);
+      const stats = await client.blockchain.getBlockStats(blockHash, ["avgfee", "height"]); 
+      
+    expect("height" in stats).toBeTruthy();
+    expect("avgfee" in stats).toBeTruthy();
+    expect(stats.height).toBeGreaterThanOrEqual(1);
+    expect(Object.keys(stats).length).toEqual(2);
+    
+    })
+  })
+  describe('getBestBlockHash', () => {
+      test('should Get hash of best block and return a string', async () => {
+        const bestBlockHash = await client.blockchain.getBestBlockHash();
+        expect(bestBlockHash).toBeTruthy();
+      })
+      
+  })
+  
```

Comment:
1. auto format properly
2. setup your workspace properly, husky should have picked this up

Reply:
Formatted properly.  I used an external tool for the commit, that is why husky did not pick it up.